packages/dds/matrix/package.json

```diff
@@ -59,6 +61,7 @@
     "temp-directory": "nyc/.nyc_output"
   },
   "dependencies": {
+    "@fluid-tools/benchmark": "^0.43.0",
```

Comment:
Let's move this to `devDependencies`, since it's only necessary to run the unit tests during development.